flixel/FlxCamera.hx

```diff
@@ -1652,6 +1660,15 @@ class FlxCamera extends FlxBasic
 		}
 		return this.visible = visible;
 	}
+	
+	private function get_zoomOffsetX():Float
```

Comment:
i'd rather set values of `zoomOffsetX` and `zoomOffsetY` when you change size or zoom of the camera, instead of calculation value every time you try to get it.